style.css

```diff
@@ -0,0 +1,123 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 50px;
+  background: #3c3a39;
+}
+
+.logo {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  position: relative;
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 0%;
+  color: white;
+}
+
+.menu-btn {
+  position: relative;
+  top: 10%;
+  right: 30px;
+}
+
+.menu {
+  width: 35px;
+  height: 5px;
+  background-color: white;
+  margin: 6px 0;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  top: 50px;
+  left: 0;
+  background-image: url("background.png");
+  background-repeat: no-repeat;
+  background-position: right 0 bottom -50px;
+  background-size: 35%;
+}
```

Comment:
- [ ] Kindly make sure that your design match the Figma template design as much as possible (e.g. font, colors, images, tests, margins)
Your design
![Capture](https://user-images.githubusercontent.com/85095702/158913726-caca3deb-badd-45b6-af50-67a9a6156ed4.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/158913741-09dba906-c23e-478c-9bf2-c8d5929edb89.PNG)